cpu/stm32l1/include/cpu_conf.h

```diff
@@ -72,6 +72,20 @@ extern "C" {
 #define CPU_FLASH_BASE                  FLASH_BASE
 /** @} */
 
+/**
+ * @brief   Flash page configuration
+ * @{
+ */
+#if defined(CPU_MODEL_STM32L152RE)
+#define FLASHPAGE_SIZE      (256U)
```

Comment:
these define are not self doxygen documented

Reply:
Are these required? I don't see any example in other boards...